foundations/html_css/flexbox/project-landing-page.md

```diff
@@ -4,7 +4,7 @@ For this project you'll be creating an entire web page from a design we'll provi
 
 The design we're providing you comes in the form of 2 images: one is an image of the complete website, and one has some details about some of the fonts and colors we've used.
 
-Do _not_ be afraid to use google or go back to previous lessons to look something up. **Real-life, professional developers use google _constantly_ for things that they have been doing for years.** At this point it is not expected that you will have everything memorized, so don't worry about it. Additionally, there are a few small details that you may not have encountered in our lessons yet. _This is by design._ These details are minor, and easily searched (e.g. google `css rounded corners`).
+Do _not_ be afraid to use google or go back to previous lessons to look something up. **In Real-life, professional developers use google _constantly_ for things that they have been doing for years.** At this point it is not expected that you will have everything memorized, so don't worry about it. Additionally, there are a few small details that you may not have encountered in our lessons yet. _This is by design._ These details are minor, and easily searched (e.g. google `css rounded corners`).
```

Comment:
In this example, "Real-life, professional developers", "Real-life" is an adjective. Here is another example of this use: "real-life heroes that changed the world...".

If you want to make this change, "Real-life" should be changed to "In real life" without the dash.

Reply:
Thank you for your feedback. I fixed the error.